Library/Formula/agda.rb

```diff
@@ -51,38 +58,61 @@ def install
     cabal_clean_lib
 
     if build.with? "stdlib"
-      resource("stdlib").stage prefix/"agda-stdlib"
+      resource("stdlib").stage lib
```

Comment:
Yeh, so this is the reason: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/35374/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_agda/

These files need to be deleted or moved somewhere other than `lib`.